requirements_test_all.txt

```diff
@@ -23,6 +23,9 @@ PyQRCode==1.2.1
 # homeassistant.components.rmvtransport
 PyRMVtransport==0.3.2
 
+# homeassistant.components.switchbot
+PySwitchbot==0.10.0
+
```

Comment:
The generate requirements script hashes out "PySwitchbot==0.10.0". The config_flow tests require this import.

Config flow uses the api to test connection to device. I also test those lines in the config_flow test by mocking the pyswitchbot api.

Reply:
pySwitchbot requires bluetooth/bluepy, and is therefore cannot be used for that. It will be automatically commented out for that reason.

Reply:
@ frenck,

Found a workaround by using pygatt to test connection to device. (same library as hass Bluetooth LE component). 

Thanks.